README.md

```diff
@@ -34,6 +34,8 @@ Add in your .csproj under `<PropertyGroup>`:
  <EmbedUntrackedSources>true</EmbedUntrackedSources>
 ```
 
+or use `/p:EmbedUntrackedSources=true` for `msbuild` or `dotnet`
+
```

Comment:
What scenario would this be better than using the property in the csproj directly?

Reply:
Enabling multiple projects or only updating the build script. 


For example: fix only in build script: https://github.com/CharsetDetector/UTF-unknown/pull/120

Reply:
Example: patching many files (OK, we needed to patch the csproj for the sourcelink already)

https://github.com/NLog/NLog/pull/4374

Reply:
If there's multiple files, why not use a `Directory.Build.props` for those instead of putting it in each `csproj`?

Reply:
That's indeed also an option. But as we already doing `/p:ContinuousIntegrationBuild=true`, it far more easier to make it `/p:ContinuousIntegrationBuild=true /p:EmbedUntrackedSources=true` IMO.

Reply:
@clairernovotny maybe I missed the goal of this repo. I assumed it was for motivating and making it easy to get the 
deterministic builds working. 

The `/p:EmbedUntrackedSources=true` helped my to get things working, but maybe because I'm not a MSBuild hero. 

I'm happy it helped my in getting this working for other packages, for example

https://github.com/App-vNext/Polly/pull/839 and  https://github.com/nunit/nunit/pull/3810

Reply:
The goal is to be opinionated and show people how to get reproducible builds working. Having those properties n command lines makes it easier to forget them/have them changed.

This is also now in the `DotNet.ReproducibleBuilds` package that can be added to your project.